lib/diskpart.ts

```diff
@@ -37,7 +37,7 @@ interface ExecResult {
 const execFileAsync = async (
```

Comment:
Why do we need `execFileAsync`? Doesn't a `util.promisify()` around `execFile()` do the same thing without this extra code? This approach is described in the [function doc](https://nodejs.org/docs/latest-v16.x/api/child_process.html#child_processexecfilefile-args-options-callback).

Reply:
It was there long ago (i don't know the reason), you may refactor it if it works.